### ISSUE TITLE
VerboseProcess.java:179-183: When VerboseProcess is closed, we...

### ISSUE THREAD

**Issue by dmarkov** [jcabi/jcabi-log#46]
Puzzle `38-9367cb1e` in `src/main/java/com/jcabi/log/VerboseProcess.java:179-183` has to be resolved: When VerboseProcess is closed, we should also shut down the monitor threads and prevent them trying to obtain the output from the process. It should be done before destroy() is called. See the following for more details: https://github.com/jcabi/jcabi-log/issues/38 http://www.java2s.com/Code/Java/Threads/Thesafewaytostopathread.htm

If you have any technical questions, don't ask me, submit new tickets instead

**Comment by wentwog** [comment]
@dmarkov please review PR #49

**Comment by dmarkov** [comment]
@wentwog just a reminder that you're working with this task for 12 days already; as a reminder, don't forget to remove the `@todo` puzzle from the source code; -30 added to your rating, current score is: [+35](http://www.netbout.com/b/35104?open=rating)

**Comment by dmarkov** [comment]
@wentwog this task is not yours any more (took too long), please stop. -60 added to your rating, at the moment it is: [+20](http://www.netbout.com/b/35104?open=rating)

**Comment by wentwog** [comment]
@dmarkov I have opened a pull request for this task 5 days ago. And no one has reviewed it yet.
So, the task is awaiting the review.
Why you withdraw the task in this situation?

**Comment by yegor256** [comment]
@dmarkov I think we should assign @wentwog back to this issue

**Comment by dmarkov** [comment]
> @dmarkov I think we should assign @wentwog back to this issue

@yegor256 yes, @wentwog is already assigned to this task, please go ahead!

**Comment by wentwog** [comment]
@dmarkov since the task withdrawal is reverted, I think, you can also revert the subtraction of 60 points.

**Comment by dmarkov** [comment]
@jenya please pick this up, and keep in mind [these instructions](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html). Any technical questions - ask right here; Budget here is **30 mins** (keep [this principle](http://www.xdsd.org/2014/04/17/how-xdsd-is-different.html) in mind)

**Comment by wentwog** [comment]
@dmarkov I don't understand what you are doing.
You've written that you canceling your prevoius decision to take this task from me:
> @wentwog is already assigned to this task, please go ahead!

**Comment by wentwog** [comment]
@dmarkov the PL is merged.
Please, close the ticked. 
And, also, please check whether the 60 pts subtracted were returned.

**Comment by yegor256** [comment]
@dmarkov hey, please assign @wentwog and don't take him away from this task :)

**Comment by dmarkov** [comment]
> @dmarkov hey, please assign @wentwog and don't take him away from this task :)

@yegor256 done, @wentwog is a new assignment at the task, please go on. whoever worked with this task before, please stop right now

**Comment by wentwog** [comment]
@dmarkov I would appreciate if you read and respond to the messages I address to you.

Now we have a situation when you ignore what I write to you; and to get the reaction from you, @yegor256 has to write here.
Let's communicate directly, since you are the task reporter and I am the task solver.

You told me to go on the task, but message above I had already told you that the pull request is closed and the changes are merged into the master branch.

So, again, please
1) Return the 60 points, you mistakenly subtracted.
2) Close the ticked since the PL is merged.
3) Assign additional points to me for the task completion, as we always do in our work process.

**Comment by wentwog** [comment]
@dmarkov please, respond. This ticket awaits your response.

**Comment by dmarkov** [comment]
I'm closing the issue...

**Comment by dmarkov** [comment]
@wentwog Thank you, I've added **30 mins** to your account in payment/transaction "`50616592`"... +30 to your rating, your total score is [+110](http://www.netbout.com/b/35104?open=rating)

**Comment by wentwog** [comment]
@dmarkov thanks, please, also respond to '1)' item in my message above.

**Comment by yegor256** [comment]
@wentwog according to our [policy](http://www.teamed.io/policy.html), there is only one person responsible for a task. If the task is not closed, for any (!) reason, it's your fault. Even if you submitted a pull request and it was not reviewed. I understand that this may sound counter intuitive at the beginning, but that's our philosophy.
